tools/ci_build/build.py

```diff
@@ -493,13 +493,16 @@ def run_onnx_tests(build_dir, configs, onnx_test_data_dir, provider, enable_para
           cmd.append(model_dir)
         if os.path.exists(onnx_test_data_dir):
           cmd.append(onnx_test_data_dir)
-        run_subprocess([exe] + cmd, cwd=cwd)
+        
         if enable_parallel_executor_test:
+          run_subprocess([exe] + cmd, cwd=cwd)  
           if provider == 'mkldnn':
             #limit concurrency to 1
             run_subprocess([exe,'-x', '-c', '1', '-j', '1'] + cmd, cwd=cwd)
           else:
             run_subprocess([exe,'-x'] + cmd, cwd=cwd)
+        else:
+          run_subprocess([exe, '-c', '1', '-j', '1'] + cmd, cwd=cwd)
```

Comment:
Why adding '-c 1 -j 1'? if it's needed for x86 build, can we restrict it only for x86 ?